dog-blog-page3.html

```diff
@@ -0,0 +1,76 @@
+<!--Adopt-a-Dog's blog Page 3 -->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt-a-Dog</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="doggystyle.css">
+  </head>
+  <body>
+    <header>
+      <div>
+        <img src="images/adoptadog-logo.svg" width="500" alt="Adoptadog logo with white dog silhoutte">
+        <h3>Seattle's Premier Dog Adoption Agency</h3>
+      </div>
+      <div class=checkouticon>
+         <img src="images/cart-icon.svg" width="40" alt="cart icon">
+          <h4><a href="dog-checkout-page4.html" class="checkbutton">Checkout</a><h4>
+        </div>
+      </header>
+      <nav>
+        <ul class="main-navigation">
+          <li><a href="index.html" class="topnav">Home</a></li>/
+          <li><a href="show-dogs-page2.html" class="topnav">Dogs</a></li>/
+          <li><a href="dog-checkout-page4.html" class="topnav">Adopt</a></li>
+        </ul>
+      </nav>
+      <main>
+         <div>
+           <h1>Adoptadog DogaBlog</h1>
+          </div>
+        <article class="main-blog-para">
```

Comment:
make sure to use full words for your classes `main-blog-para` doesn't mean anything to me. For these simply `dog-blog` would be appropriate